src/components/booksform.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Let move this file into the `container` folder, for any of our component connecting to the redux store are kept inside the `container` folder.
![image](https://user-images.githubusercontent.com/31897434/74389667-3e925c80-4dff-11ea-8e0c-d533c73b0109.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/